### PR TITLE
feat: add web starter bb-1

### PR DESCRIPTION
**Pull request by WilhelmWesser**

### CODE REVIEW

File: .ls-lint.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ls:
+  .dir: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
+  .css: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
+  .ts: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
+  .tsx: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
```
Comment (fshabanov) [inline .ls-lint.yml:5]: We can use `kebab-case`, instead of regex
https://ls-lint.org/1.x/configuration/the-rules.html

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
+# BeBalance
+
+Track your progress for all your courses
```
Comment (fshabanov) [inline readme.md:3]:
```suggestion
```
This is not about our project

File: apps/backend/package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+	"name": "backend",
+	"type": "module",
+	"version": "1.0.0",
```
Comment (fshabanov) [inline apps/backend/package.json:4]: Add `engines`

File: apps/frontend/index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<link rel="icon" type="image/svg+xml" href="/vite.svg" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Vite + React + TS</title>
```
Comment (fshabanov) [inline apps/frontend/index.html:7]:
```suggestion
		<title>BeBalance</title>
```

File: .ls-lint.yml (hunk 1 above)
Comment (WilhelmWesser) [reply .ls-lint.yml:5]: I do not think it is an option. These RegExp also covers case when the file is named like a-b.c.ts but kebab-case will not

File: eslint.config.js [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+import importPlugin from "eslint-plugin-import";
+import jsdoc from "eslint-plugin-jsdoc";
+import perfectionist from "eslint-plugin-perfectionist";
+import sonarjs from "eslint-plugin-sonarjs";
+import unicorn from "eslint-plugin-unicorn";
+import globals from "globals";
+
+const JS_MAX_PARAMS_ALLOWED = 3;
+
+/** @typedef {import("eslint").Linter.FlatConfig} */
```
Comment (fshabanov) [inline eslint.config.js:14]:
```suggestion
/** @typedef {import("eslint").Linter.Config} */
```
I believe FlatConfig is deprecated

**Review by fshabanov — APPROVED** [review]

**Pull request merged**